### DISCUSSION TITLE
Error building translations

### DISCUSSION THREAD

**Discussion by userkilled** [zyrouge/symphony#107] [General]
> @zyrouge/symphony-cli@0.0.0 i18n:build /home/symphony
> ts-node ./cli/i18n/build.ts

/home/symphony/node_modules/yaml/dist/compose/composer.js:33
                if (prelude[i + 1]?.[0] !== '#')
                                   ^

SyntaxError: Unexpected token .
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.require.extensions.(anonymous function) [as .js] (/home/symphony/node_modules/ts-node/src/index.ts:1608:43)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/home/symphony/node_modules/yaml/dist/index.js:3:16)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @zyrouge/symphony-cli@0.0.0 i18n:build: `ts-node ./cli/i18n/build.ts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @zyrouge/symphony-cli@0.0.0 i18n:build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

-----
To be honest, I don’t understand anything. What the problem is. It is possible that the version is not suitable, I have npm v 6.14.4

**Comment by zyrouge** [comment]
~~Seems like it failed to parse the yaml file. You have an invalid `.` somewhere in your file.~~

**Comment by zyrouge** [comment]
You can open a draft PR if you need help.

**Comment by userkilled** [comment]
https://github.com/eemeli/yaml/issues/394

need version node =>14
(

https://github.com/nodesource/distribution

**Comment by userkilled** [comment]
my node version 10.x

**Comment by zyrouge** [comment]
Update to v18.x version.

**Comment by userkilled** [comment]
and I still had to clone the project again, t.to. lock_file interfered

**Comment by userkilled** [comment]
npm v 9.5.0

**Comment by zyrouge** [comment]
> and I still had to clone the project again, t.to. lock_file interfered

Check if,
- Node version is 18.x (NPM version doesn't matter if it's up-to-date)
- You install the dependencies.

Not sure what you mean by that.

**Comment by userkilled** [comment]
until that moment I just collected it on the old version. and 18 tried to clean it...



By the way, I manage to collect the project, but for this I need to constantly open the SDK storage settings .
 
Well now it is going, but I could not integrate the translation for Russian. I had to translate the en.yaml file itself
I found that the font becomes what is used in the system. (But that is not a problem.... and features))))

**Comment by zyrouge** [comment]
I guess this is no longer an issue? Feel free to re-open this if it is.
